mm2src/coins/lp_coins.rs

```diff
@@ -24,14 +24,15 @@
     forgetting_references,
     forgetting_copy_types,
     clippy::swap_ptr_to_ref,
-    clippy::forget_non_drop
+    clippy::forget_non_drop,
+    clippy::doc_lazy_continuation,
+    clippy::result_large_err,
```

Comment:
If we are gonna use this globally then it should be removed from `custom_token_validations` function